app/presenters/dttp/trainee_presenter.rb

```diff
@@ -39,28 +45,23 @@ def placement_assignment_params
       }
     end
 
-  private
-
-    def gender_code
-      trainee.male? ? MALE_GENDER_CODE : FEMALE_GENDER_CODE
-    end
-
-    def dttp_programme_subject_id
-      Dttp::CodeSets::ProgrammeSubjects::MAPPING.dig(trainee.subject, :entity_id)
-    end
-
-    def dttp_course_phase_id
-      Dttp::CodeSets::AgeRanges::MAPPING.dig(trainee.age_range, :entity_id)
+    def uk_degree_params
+      {
+        "dfe_SubjectofUGDegreeId@odata.bind" => "/dfe_jacses(#{dttp_degree_subject_id})",
+        "dfe_AwardingInstitutionId@odata.bind" => "/accounts(#{dttp_degree_institution_id})",
```

Comment:
Is this the correct entity set - `accounts`? I sort of thought that was the providers that have users using the system. I mean, that have a lot of crossover with awarding institutions but presumably there are schools and other non-degree awarding things in there.

Reply:
I wrote a script to search all entities in DTTP for say `443e2cff-6f42-e811-80ff-3863bb3640b8` (Aberystwyth University). It only found the entity ID in `/accounts`. The CSV file `/accounts` has 52531 entries.